systemtest/src/test/java/io/strimzi/systemtest/operators/NamespaceDeletionRecoveryST.java

```diff
@@ -46,6 +46,7 @@
     private static final Logger LOGGER = LogManager.getLogger(NamespaceDeletionRecoveryST.class);
 
     private String storageClassName = "retain";
+    private static final String CLUSTER_NAME = "my-cluster";
```

Comment:
Why the static name? Wasn't this a problem in some previous tests and recently changed?
@Frawless ^^

Reply:
Because the value is by default changed before every test and it breaks it afaiu

Reply:
Yes, exactly.